iac/terraform/aws/web-client/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "environment" {
+  type        = string
+  description = "Deployment environment"
+}
+
+variable "bloodconnect_domain" {
+  description = "The domain name for the web client"
```

Comment:
i belive this is the root domain and it's not just for web client. It will be used for API endpoints and other purposes. Let's update documentation

Reply:
Done